static/app/views/settings/project/tempest/CredentialRow.tsx

```diff
@@ -19,6 +33,36 @@ export function CredentialRow({credential}: {credential: TempestCredentials}) {
       <Flex align="center">
         {credential.createdByEmail ? credential.createdByEmail : '\u2014'}
       </Flex>
+
+      <Flex align="center">
```

Comment:
usually the actions column has a title and the buttons are right aligned, shouldn't we do the same?

example:

![image](https://github.com/user-attachments/assets/8f02e655-615b-46fa-ab26-1b7062234f8f)

tempest UI:

![image](https://github.com/user-attachments/assets/59e315b6-ca86-4c93-ba60-819c5a8604ab)